README.md

```diff
@@ -24,26 +24,61 @@ npm run lint
 
 ### Customize configuration
 
-See [Configuration Reference](https://cli.vuejs.org/config/).
+See [Configuration Reference](https://cli.vuejs.org/config/) for Vue options.
+
+## Optional: Setup website backend server (and authenticate as admin)
+By default the website you started using `yarn run serve` will connect to the productive
```

Comment:
Technically I think it will connect to whatever the last commit to `env.js` was 😃